zerver/lib/validator.py

```diff
@@ -389,6 +389,22 @@ def check_url(var_name: str, val: object) -> str:
         raise ValidationError(_("{var_name} is not a URL").format(var_name=var_name))
 
 
+def check_capped_url(max_length: int) -> Validator[str]:
+    def validator(var_name: str, val: object) -> str:
+        # Ensure val is a string and length of the string does not
+        # exceed max_length.
+        s = check_capped_string(50)(var_name, val)
```

Comment:
I think you wanted to pass `max_length` here, right?

Reply:
Oops, fixed it.